workflow-templates/library-checks-and-release.yml

```diff
@@ -0,0 +1,57 @@
+name: Library checks & release
+
+on:
+  push:
+    branches:
+      - master
+      - next
+      - alpha
+      - beta
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+  SURGE_LOGIN: ${{ secrets.SURGE_LOGIN }}
+  SURGE_TOKEN: ${{ secrets.SURGE_TOKEN }}
+
+jobs:
+  main:
+    name: Run checks and deploy
+    runs-on: [self-hosted, fear]
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

Comment:
Current version is 15... can we bump it, or are there restrictions from other services/libraries?

Reply:
we should actually start moving to the active LTS (v14), as v15 is not guaranteed to be stable: https://github.com/nodejs/Release

but we're currently at 12 almost everywhere, so let's keep it like this for now. and let's tackle upgrading the whole org separately.